### PR TITLE
feat(unlock-app) - remove seconds from event details pages

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

- [x] end time is always after start time
- [x] end date is always after/equal start date
- [x] remove seconds from event page details 

**Error when start/end date are the same end time should be later/equal start time** 

Screenshot 2023-05-10 at 11 38 22

**Error when end date if before start date** 

Screenshot 2023-05-10 at 12 56 18

**Disable dates before the start date** 

Screenshot 2023-05-10 at 12 01 43

**Remove seconds from event page details** 

Screenshot 2023-05-10 at 12 11 35

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/content/event/Form.tsx [hunk 1]
```diff
@@ -290,16 +314,27 @@ export const Form = ({ onSubmit }: FormProps) => {
...
+                        required: {
+                          value: true,
+                          message: 'This value is required',
+                        },
+                      })}
                       type="time"
+                      min={minEndTime}
                       label="End time"
+                      error={
+                        // @ts-expect-error Property 'event_end_time' does not exist on type 'FieldError | Merge<FieldError, FieldErrorsImpl<any>>'.
```
Comment (searchableguy) [inline unlock-app/src/components/content/event/Form.tsx:335]: You should be able to type this.

Comment (kalidiagne) [reply unlock-app/src/components/content/event/Form.tsx:335]: is already defined in `NewEventForm` but this still gives ts error for nested parameters like `metadata.ticket.event_start_date`

**Comment by julien51** [comment]
Great! Thanks! I will hotfix this!

**Comment by patrickworkman** [comment]
Seconds still appear on event details pages. Do organizers need to create another EDP to remove them? 

[Example EDP with seconds](https://app.unlock-protocol.com/event?s=web3innovators)

**Pull request merged**